src/ResourceManager/AzureBatch/documentation/current-breaking-changes.md

```diff
@@ -38,4 +38,95 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-## Current Breaking Changes
\ No newline at end of file
+## Current Breaking Changes
+## Release 5.0.0 - November 2017
```

Comment:
Please don't do this, please leave the upcomnign breaking changes as is, we will move all of these automatically at the end of the release.

Reply:
Fixed